pkgs/games/space-cadet-pinball/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, fetchzip
+, cmake, SDL2, SDL2_mixer
+, unrar-wrapper, makeWrapper
+}:
+
+let
+  assets = (fetchzip {
+    url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
+    sha256 = "sha256-fC+zsR8BY6vXpUkVd6i1jF0IZZxVKVvNi6VWCKT+pA4=";
+    stripRoot = false;
+  }).overrideAttrs (old: {
+    nativeBuildInputs = old.nativeBuildInputs ++ [ unrar-wrapper ];
+  });
+in
+stdenv.mkDerivation rec {
+  pname = "SpaceCadetPinball";
+  version = "389122182e34f9b37d5476b797bb7b7b4d217f51";
```

Comment:
Please use the `unstable-YYYY-MM-DD` format.

Reply:
BTW, commit https://github.com/k4zmu2a/SpaceCadetPinball/commit/de13d4e326b2dfa8e6dfb59815c0a8b9657f942d adds a Linux desktop file and https://github.com/k4zmu2a/SpaceCadetPinball/commit/389122182e34f9b37d5476b797bb7b7b4d217f51 fixes a typo, so I think they're better commits to pick up.